tests/integration/rdb.tcl

```diff
@@ -259,6 +272,21 @@ start_server {overrides {save ""}} {
                 fail "COW info wasn't reported"
             }
 
+            # assert that $current_processed_keys is not greater than keys on start.
+            assert_morethan_equal $keys_on_bgsave_start $current_processed_keys
```

Comment:
we expect it to be equal (never greater)

Reply:
keys_on_bgsave_start are the total keys in DB, It should be always greater or equal isn't it?

Reply:
ohh, i forgot that the test (when run without `--accurate`) can abort before the bgsave is completed.